bridge/engine/EngineStreamDirector.h

```diff
@@ -336,9 +336,10 @@ class EngineStreamDirector
         const size_t numRecordingStreams)
     {
         const auto quality = getQualityLevel(ssrc);
+        const auto highestAvailableQuality = highestActiveQuality(senderEndpointIdHash, ssrc);
 
-        // We server low and mid for unpinned according to the ConfigLadder.
-        if (highQuality != quality && isUnpinnedQualityUsed(quality, senderEndpointIdHash, isSenderInLastNList))
+        // We serve low and mid for unpinned according to the ConfigLadder.
+        if (isUnpinnedQualityUsed(quality, highestAvailableQuality, senderEndpointIdHash, isSenderInLastNList))
```

Comment:
This is my suggestion for readability 
```
cont bool isPinnedQuality = quality == highestAvailableQuality;
if (!isPinnedQuality) {
    return isUnpinnedQualityUsed(quality, highestAvailableQuality, senderEndpointIdHash, isSenderInLastNList);

}

Reply:
See previous comment.
I'm going to refactor the code to make it more readable and intentions clear. I see that now it's quite confusing,

Reply:
The name creates some confusion. Is more about if non high quality is used instead of "unpinned" quality